### ISSUE TITLE
Upgrade Berkeley DB

### ISSUE THREAD

**Issue by robertdale** [JanusGraph/janusgraph#652]
https://docs.oracle.com/cd/E17277_02/html/changelog.html

> Upgrading from JE 7.3 or earlier
> 
> No file format changes were included in JE 7.4 and there are no file format compatibility issues when upgrading from JE 7.3.
> 
> Changes in 7.4.5
> 
> Fixed an internal deadlock between the following internal classes: ExpirationProfile, IN, FileSelector. This deadlock occurred very rarely (only once in our testing). It did not cause a persistent problem -- restarting the process was a safe workaround. [#25613] (7.4.1)
> 
> EnvironmentConfig.CLEANER_FORCE_CLEAN_FILES has been made mutable. [#25821] (7.4.2)
> 
> The OperationResult.isUpdate() method has been added for distinguishing inserts and updates performed by a Put.OVERWRITE operation. [#25882] (7.4.2)
> 
> Fixed unsafe file deletion issue in network restore. If the network restore was performed on a node without closing the environment, the deletion of obsolete log files was considered an unsafe operation that caused the restore to fail. The error message was:
>   com.sleepycat.je.EnvironmentFailureException:../env Log file 00000000.jdb
>   was deleted unexpectedly. LOG_UNEXPECTED_FILE_DELETION: A log file was
>   unexpectedly deleted, log is likely invalid. Environment is invalid and must
>   be closed. Originally thrown by HA thread: REPLICA 3(-1) 
> [#25834] (7.4.3)
> 
> Logging of internal nodes (INs) has been reduced when deleting many records in a contiguous key range. [#25939] (7.4.3)
> 
> Fixed a bug that could have caused duplicate records to be returned via the iterator() method of the DPL and Collections API. The iterator reads records in batches, and if a record at the end of the last batch was deleted by another thread, fetching the next batch could have read (and later returned via the iterator) duplicate records, depending on thread timing. [#25976] (7.4.3)
> 
> Fixed a bug that prevented LogOverwriteException from being thrown when assertions were disabled. LogOverwriteException is thrown to prevent creation of an invalid backup, although this can only happen in rare cases and only on an HA replica node. See the LogOverwriteException javadoc for more information. [#25989] (7.4.4)
> 
> Fixed a bug that caused the following assertion to fire when using a Database in deferred-write mode (DatabaseConfig.setDeferredWrite(true)) for which data was previously written in normal (non-deferred-write) mode.
> java.lang.AssertionError
>   com.sleepycat.je.tree.BIN.shouldLogDelta(BIN.java:1927)
>   ...
> [#25999] (7.4.4)

**Comment by mbrukman** [comment]
Closing as the linked PR was merged.
